.github/CODEOWNERS

```diff
@@ -370,6 +370,7 @@ packages/kbn-ftr-common-functional-services @elastic/kibana-operations @elastic/
 packages/kbn-ftr-common-functional-ui-services @elastic/appex-qa
 packages/kbn-ftr-screenshot-filename @elastic/kibana-operations @elastic/appex-qa
 packages/kbn-gen-ai-functional-testing @elastic/appex-ai-infra
+packages/kbn-genai-common @elastic/response-ops
```

Comment:
Regarding the naming, I'd recommend to have a dedicated name like `kbn-inference-endpoint-ui-common`, or something similar, because we already have a set of genai related packages and plugins, and also x-pack/packages/ai-infra/inference-common ...
As for the code owners, we should extend the list with @elastic/appex-ai-infra, @elastic/obs-ai-assistant @elastic/security-generative-ai

Reply:
Agreed, we have already too many "gen-ai-common" things, unfortunately we probably need to be slightly more specific here.

Reply:
@YulNaumenko: Should we update the owners to only `ml-ui` as they will own both `inference management ui` and `AI connector`?